esp/smc/SMCLib/TpCommon.cpp

```diff
@@ -97,3 +97,12 @@ extern TPWRAPPER_API IStringIterator* getContainerTargetClusters(const char* pro
     }
     return ret.getClear();
 }
+
+extern TPWRAPPER_API bool validateNetAddressRequest(const char* netAddressReg, bool ipReq, IConstTpMachine& tpMachine)
```

Comment:
Minor, but I would change the name of this function, you are matching or comparing the value, not validating it.  

@ghalliday Doesn't have to prevent merging though if we want this to make it into 8.2.